contracts/interfaces/IERC4626Strict.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.17;
+
+import "./IERC20.sol";
+
+interface IERC4626Strict {
+  function asset() external view returns (address);
```

Comment:
you do not need dedicated interface, use IERC4626.sol

Reply:
Done